steps/get-npm-package-version/1.0.0/step.yml

```diff
@@ -0,0 +1,41 @@
+title: Get npm package version
+summary: |
+  Sets an environment variable with the npm package version
+description: |
+  Sets environment variable `$PACKAGE_VERSION` with the value found in the "version" field of package.json, for use in later steps
```

Comment:
Typo in `$PACKAGE_VERSION` => `$NPM_PACKAGE_VERSION`

Reply:
Whoops, good catch! I originally had it as `$PACKAGE_VERSION` everywhere then updated it to `$NPM_PACKAGE_VERSION` to be more explicit, but I missed the description and README.